care/facility/migrations/0128_facilityrelatedsummary.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.11 on 2020-06-29 15:26
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
```

Comment:
Codacy found an issue: [standard import "import uuid" should be placed before "import django.contrib.postgres.fields.jsonb"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5811718)